beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -35,6 +35,7 @@ func New() *ForkChoice {
 		nodeByPayload:                 make(map[[fieldparams.RootLength]byte]*Node),
 		slashedIndices:                make(map[types.ValidatorIndex]bool),
 		pruneThreshold:                defaultPruneThreshold,
+		receivedBlocksLastEpoch:       [32]types.Slot{},
```

Comment:
Magic number?

Reply:
slots per epoch, might be an overkill but I can add a constant